functions/models/src/functions/createInvitation.ts

```diff
@@ -16,9 +16,6 @@ export const createInvitationInputSchema = z.object({
 })
 export type CreateInvitationInput = z.input<typeof createInvitationInputSchema>
 
-export const createInvitationOutputSchema = z.object({
-  id: z.string(),
-})
-export type CreateInvitationOutput = z.output<
-  typeof createInvitationOutputSchema
->
+export interface CreateInvitationOutput {
```

Comment:
It doesn't make sense to create output schemas. Input needs validation, but output can be safely checked using TypeScript interfaces only.